safe/impact_function/impact_function.py

```diff
@@ -1766,6 +1765,7 @@ def post_process(self, layer):
         # Set the layer title
         layer_title(layer)
 
+        from safe.definitions.post_processors import post_processors
```

Comment:
> So we can't import post_processors at the beginning of file as usual ?

I think yes if we want to have a direct effect after changing needs profile, `post_processors` value has to be changed to apply new minimum needs post processors. I'll write a comment for it.